### ISSUE TITLE
Crash when forming a tank

### ISSUE THREAD

**Issue by PokeMaserJ** [Lordmau5/FFS-v1.X#68]
**FancyFluidStorage Version**: Fancy Fluid Storage-1.9.4-1.6.4-b28

**MinecraftForge Version**: 12.18.1.2014 (MC 1.10.2)

**Client or Server?**:
- [x] Client
- [ ] Server

**Crash Report (via. [Pastebin](http://pastebin.com/))**: http://pastebin.com/QAJxgM6D 

**What happens / happened?**: Made a tank , instant crash to desktop

**How can we reproduce it?**: Make a Tank

**Comment by marcin212** [comment]
**FancyFluidStorage Version**:
FFS v1.6.4-b28

**MinecraftForge Version**:
12.18.1.2046

**Client or Server?**:
- [ ] Client
- [x] Server

**Crash Report (via. [Pastebin](http://pastebin.com/))**:
http://pastebin.com/PtT7i7ua

**What happens / happened?**:
server crash

**How can we reproduce it?**:
Make a Tank from TiCon Glass

**Comment by PokeMaserJ** [comment]
Is there anything I can do as a "player" to help you guys fix this?

**Comment by Lordmau5** [comment]
Yes.

Wait for Minecraft 1.11 to be released and for FFS v2.0 to be done, that's the only thing I can offer.

The current state the mod is in is utter garbage and shouldn't have coded it like this.
The new state will be much, MUCH less prone to crashes and will have way better support when it comes to other mods' stuff.

TL;DR: I'm not gonna fix any bugs that arise for the current version of FFS and am not going to bring out an update / bugfix for any version that is below Minecraft 1.11.

**Comment by SubsterNL** [comment]
Why not close down the mod completely. Looking at the issue reports you rather close and kill the reports than doing something. People do spend time posting issues. Bit rude for you to simply throw it back in their faces.

You seem to be new at coding and such. Perhaps start some off-line projects?

**Comment by Lordmau5** [comment]
@SubsterNL How about you spend your time on other Github repos if my repo doesn't seem *worthy* enough?

Honestly though, FFS 2.0 is close to a release. The current code-base is horrible due to the fact that I do block-replacement and fake worlds. Crashes are sure to happen with that, obviously.

Other than that, my response to you posting stuff on a close issue:
![nope](http://i.imgur.com/fklmi.gif)

**Comment by jaredlll08** [comment]
Having coded a multiblock myself, I have to say that no new modder could do anything that @Lordmau5 has done, multiblocks are very complex concepts and not for the faint of heart.
A new modder would not have been able to do anything close to what @Lordmau5 has done, heck, I couldn't even get my multiblock to stop lagging and it wasn't even close to what FFS does.
